2269/2/raper.rb

```diff
@@ -0,0 +1,32 @@
+# Class that represents each Rap Singer.
+# :reek:Attribute
+# :reek:TooManyInstanceVariables
+class Raper
+  def initialize(a_name)
+    @name = a_name
+    @words_round = 0
+    @words_battle = 0
+    @bad_words = 0
+    @battles = 0
+    @file_name = []
```

Comment:
use memoization for instance variables.

Reply:
initialiazation of instance variables has been changed to memoization